### PR TITLE
JCRVLT-776: improve test coverage for IdConflictPolicy

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

File: vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java [hunk 1]
```diff
@@ -418,28 +418,131 @@ public void testImportDupPolicyLegacy() throws RepositoryException, IOException,
...
 
+    // constants for behavior target state to be tested
+
+    private static enum TS {
+        CONFLICT_TARGET_MOVED, CONFLICT_TARGET_PRESENT, NO_CONFLICT_TARGET_GONE, NO_CONFLICT_TARGET_UNCHANGED
+    };
+
+    // make boolean expectations readable
+
+    private static Boolean ID_NEW = true;
```
Comment (kwin) [inline vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java:429]: why not encapsulating the semantics within the `enum TS`?

File: vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java [hunk 2]
```diff
@@ -418,28 +418,131 @@ public void testImportDupPolicyLegacy() throws RepositoryException, IOException,
         assertNull(duplicateNode);
     }
 
+    // constants for behavior target state to be tested
+
+    private static enum TS {
```
Comment (kwin) [inline vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java:423]: I prefer a slightly longer name

File: vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java (hunk 1 above)
Comment (reschke) [reply vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java:429]: too many variants...?

File: vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java (hunk 2 above)
Comment (reschke) [reply vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java:423]: ok

**Comment by reschke** [comment]
@kwin - could you please have another look?

**Review by kwin — APPROVED** [review]

**Pull request merged**